packages/tradershub/src/features/cfd/components/TradingAppCardLoader/TradingAppCardLoader.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
@hirad-deriv please create a loader directory in components and put this TradingAppCardLoader in it

Reply:
Done